essentials-framework/Essentials DM/Essentials_DM/Endpoints/DGEs/DmDge200CController.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Crestron.SimplSharp;
+using Crestron.SimplSharpPro;
+using Crestron.SimplSharpPro.UI;
+
+using Newtonsoft.Json;
+
+using PepperDash.Core;
+using PepperDash.Essentials.Core;
+using PepperDash.Essentials.Core.Config;
+using Crestron.SimplSharpPro.DeviceSupport;
+
+namespace PepperDash.Essentials.DM.Endpoints.DGEs
+{
+    /// <summary>
+    /// Wrapper class for DGE-100 and DM-DGE-200-C
+    /// </summary>
+    public class DmDge200CController : CrestronGenericBaseDevice, IComPorts, IIROutputPorts, 

```

Comment:
This could inherit from the Dge100 controller class directly, benefiting from the logic you did there.

Reply:
Fixed to inherit appropriately.